TypeCobol.Analysis/Graph/ControlFlowGraph.cs

```diff
@@ -226,7 +226,7 @@ internal void AddWrongOrderPerformThru(PerformProcedure performThru, N procedure
         /// </summary>
         /// <param name="perform">Perform node</param>
         /// <param name="recursiveJump">Recursive jump node</param>
-        internal void AddRecursivePerform(PerformProcedure perform, N recursiveJump)
+        internal void AddRecursivePerform(PerformProcedure perform, List<N> recursiveJump)
```

Comment:
There is no need to check for the existence of the perform node entry in the dictionary now, we can call `Add` each time. The second parameter should be renamed also (but I can't find a proper name yet, it is not the cycle, it's all the perform reached through the one passed as first arg).

Reply:
Done in 35d0734